rotkehlchen/externalapis/blockscout.py

```diff
@@ -195,7 +195,7 @@ def query_withdrawals(self, address: ChecksumEvmAddress) -> set[int]:
                 self.db.set_dynamic_cache(
                     write_cursor=write_cursor,
                     name=DBCacheDynamic.WITHDRAWALS_IDX,
-                    value=index_to_write,
+                    value=int(index_to_write),
```

Comment:
this is not needed. Index is an int already.
```suggestion
                    value=index_to_write,
```